client-app/src/Components/DonorListTest.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { render, screen, fireEvent, waitFor } from '@testing-library/react';
+import axios from 'axios';
+import DonorList from './DonorList'; // Adjust the import based on your file structure
+
+jest.mock('axios');
+
+describe('DonorList Component', () => {
```

Comment:
@truffer11 we have seperate tests folder for writing tests, can you please move this test suite to there? also ensure you follow the same file naming convention .

Reply:
Also there are merge conflicts